### DISCUSSION TITLE
Scale figures so they look proportional

### DISCUSSION THREAD

**Discussion by jandermoreira** [quarto-dev/quarto-cli#6588] [Q&A] [answered]
### Description

I have lots of images with variable widths to use in a book and I'd like them to look proportional. 
Since some figures are really wide, I want to scale them down so they look proportional.
So far, my attempts were unsuccessful. How to get this?

Here's a MWE trying to scale the images:

```qmd
---
project:
  type: book

book:
  title: My Title
  author: Me
  chapters:
    - index.qmd

format:
  html:
    theme: doc.css
---

# Intro

## Actual size
![](a.png)

## Scaled

![](a.png){.Scale}

![](ab.png){.Scale}

![](abc.png){.Scale}

![](abcd.png){.Scale}
```
This is the CSS file:

```css
/*-- scss:defaults --*/

.Scale {
  transform: scale(0.25);
}

These are sample images:
```
![a](https://github.com/quarto-dev/quarto-cli/assets/29583273/2c4fc22c-3591-41d7-b24c-8a1f6b6c136e)
![ab](https://github.com/quarto-dev/quarto-cli/assets/29583273/3810dc8d-b0b9-42bb-8c1c-6432480f31fe)
![abc](https://github.com/quarto-dev/quarto-cli/assets/29583273/d2af70b6-0502-4408-815b-9ce7c76ac2b3)
![abcd](https://github.com/quarto-dev/quarto-cli/assets/29583273/91f9f741-ccf9-470e-88be-64824c566464)

**Comment by jandermoreira** [comment]
Setting `fig-responsive: false` seems to work...

**Comment by mcanouil** [comment]
It's really unclear what you mean by "proportional" since Quarto keep the aspect ratio, that makes not much sense.
If you want to change the width, simply set it, see <https://quarto.org/docs/authoring/figures.html#figure-sizing>.

`fig-responsive` is about rescaling the figure based on the device on which it is observed, _i.e._, downscale on small screen, see <https://quarto.org/docs/reference/formats/html.html#figures>.

**Reply by jandermoreira** [reply]
I meant related to each other. I want all A, for example, to have the same size, but if the image is large, it bicames smaller.

Here is my actual text. I want all the squares with letters to look the same size in all figures. And I want to avoid resizing every single image...

This is what I get so far:

![image](https://github.com/quarto-dev/quarto-cli/assets/29583273/eb4e268c-57f2-4208-80c4-f09ec88705e7)


This is what I want:
![image](https://github.com/quarto-dev/quarto-cli/assets/29583273/3819dcd2-dd18-4c97-9f8f-118b31480e4d)

**Reply by mcanouil** [reply, marked as answer]
Ok, so there nothing proportional at least from the HTML/CSS perspective.
As long as you take care of generating the figures correctly and that you set their width properly, you should not have to resign on the responsive feature.
